### PR TITLE
javascript-loops - Lamberto Martinez

### PR DESCRIPTION
**Pull request by Lambpato**
Screenshot 2023-02-01 at 4 20 04 PM

### CODE REVIEW

File: javascript-loops/main.js [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+
+function getNumbersToTen() {
+  var numbers = [];
+  var currentNumber = 0;
```
Comment (shawnkost) [inline javascript-loops/main.js:12]: This should be 1

File: javascript-loops/main.js [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+  while (currentNumber < 10) {
```
Comment (shawnkost) [inline javascript-loops/main.js:14]: This should be less than or equal to. We want to include 10

File: javascript-loops/main.js [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+    numbers.push(currentNumber);
+  }
+  return numbers;
+}
+
+console.log('getNumbersToTen():', getNumbersToTen());
+
+function getEvenNumbersToTwenty() {
+  var evenNumbers = [];
+  var currentNumber = 0;
```
Comment (shawnkost) [inline javascript-loops/main.js:25]: This should be 2

File: javascript-loops/main.js [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+
+  while (currentNumber < 20) {
```
Comment (shawnkost) [inline javascript-loops/main.js:27]: This should be less than or equal to. We want to include 20

**Review by shawnkost — CHANGES_REQUESTED** [review]
A few changes requested here Lamberto!

**Review by JenBowman1 — APPROVED** [review]
👍

**Pull request merged**